Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ubuntu:20.04
+FROM ubuntu:22.04 AS build
 RUN apt update -y -q
```

Comment:
While at it, could you merge these `RUN` instructions (i.e. `RUN apt update -y -q && DEBIAN_FRONTEND=noninteractive apt-get install --no-install-recommends -y -q curl build-essential ca-certificates ...`) to reduce the number of layers (see https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#minimize-the-number-of-layers) as well?

Reply:
I've managed to club the different `RUN` steps to reduce the number of layers.